libs/dao-data/src/utils/transformers.ts

```diff
@@ -1,6 +1,57 @@
 import { votingPowerPercentage } from '@daohaus/common-utilities';
-import { QueryResult, TransformedMembership } from '..';
 import { ListMembershipsQuery } from '../subgraph/queries/members.generated';
+import { Proposal } from '../subgraph/schema.generated';
+import {
+  TransformedProposal,
+  TransformedMembership,
+  QueryResult,
+  AccountProfile,
+  BasicProfile,
+  TokenBalance,
+  DaoTokenBalances,
+} from '../types';
+import { getProposalStatus } from './proposals';
+
+export const transformProposal = (
+  proposal: Partial<Proposal> | undefined
+): TransformedProposal => {
+  if (!proposal) {
+    return {};
+  }
+  return { ...proposal, status: getProposalStatus(proposal) };
+};
+
+export const transformProfile = (
+  address: string,
+  ens: string | null,
+  profile: BasicProfile
+): AccountProfile => {
+  return {
+    address,
+    ens,
+    ...profile,
+    image:
+      profile.image?.original?.src &&
+      `https://daohaus.mypinata.cloud/ipfs/${profile.image.original.src.match(
```

Comment:
I'm gathering that this match() statement is a way to derive a Pinata enpoint from the original IPFS hash. 

Could we refactor this so it could be used again when we handle other non-profile images?

Reply:
> I'm gathering that this match() statement is a way to derive a Pinata enpoint from the original IPFS hash.
> 
> Could we refactor this so it could be used again when we handle other non-profile images?

Yeah I like that. Going to add a future refactor ticket here to address once we get another use case/image source.